index.html

```diff
@@ -1806,30 +1809,49 @@
                   </ol>
                 </li>
                 <li>If <var>previousAvailability</var> is not equal to
-                <var>newAvailability</var>, then <a>queue a task</a> to run the
-                following steps:
+                <var>newAvailability</var>, then run the following steps:
                   <ol>
-                    <li>Set <var>A</var>'s <a>value</a> property to
-                    <var>newAvailability</var>.
+                    <li>If <var>A</var> has already been exposed to ECMAScript
```

Comment:
I pushed an update to the PR that replaces "exposed to ECMAScript" with an "initialized" notion, explained in an editorial note next to the step that uses it. I'd say that's enough, but if you feel something more formal is needed, we would need to introduce an "initialized" boolean flug for each tuple in the set.